ui-admin/src/util/tableUtils.tsx

```diff
@@ -417,7 +418,7 @@ const defaultBasicTableConfig = {
 /** helper function for simple table layouts */
 export function basicTableLayout<T>(table: Table<T>, config: BasicTableConfig<T> = {}) {
   const { filterable } = { ...defaultBasicTableConfig, ...config }
-  return <table className={config.tableClass ? config.tableClass : 'table table-striped'}>
```

Comment:
Might be controversial, but since these tables were becoming more complex, striping started to look kinda bad. And at the demo, nobody seemed to notice 😉.  Upon googling it, most people seem to recommend _not_ zebra striping, so I thought I'd open it to the council to see what people's thoughts are.

Reply:
Hugely in favor of getting rid of the striped styling. I think it looks a lot cleaner like this!

Reply:
I agree it looks a bit nicer, but I find it harder to use -- my eyes have a harder time tracking across an individual row.  Maybe we pose the question to Erin to see if we can get a one-word response?  Alternatively, consider keeping the striping but getting rid of the borders around the stripes for a somewhat cleaner look:

<img width="802" alt="image" src="https://github.com/user-attachments/assets/bc447a02-af40-4a12-8b6d-84e7355e8362">

Reply:
FWIW I bumped into Erin yesterday and showed off a few of our recent UX changes. The new table styling got a fairly indifferent 🤷. It doesn't seem like there are any well defined best practices around it

Reply:
Agree on talking to erin - what prompted to me to do this now specifically is that striping looks (imo) objectively bad with the family view:
<img width="1312" alt="image" src="https://github.com/user-attachments/assets/44f6fe1f-fe72-413b-9407-6b7464e7c0ff">
There _might_ be a way to get the sub-table to group into the same row as its parent row, but it would require a non-trivial amount of experimentation (to my knowledge)